### PR TITLE
Suppress internalAstError

### PR DESCRIPTION
**Pull request by rotu**
`cppcheck` can, in some cases, construct an invalid parse tree.
http://build.ros2.org/user/rotu/my-views/view/CycloneDDS/job/Fci__nightly-cyclonedds_ubuntu_focal_amd64/lastCompletedBuild/testReport/rclcpp/cppcheck/error__internalAstError__src_rclcpp_clock_cpp_159_/

```
- rclcpp.cppcheck error: internalAstError (src/rclcpp/clock.cpp:159)
  <<< failure message
    Syntax Error: AST broken, 'if' doesn't have two operands.
  >>>
```
This error sounds like a syntax error, but is in fact, not. (cppcheck doesn't seem to even find the syntax error in `if(1,1){}`)
This commit causes such errors to be silently ignored, as they are not actionable.
Signed-off-by: Dan Rose <dan@digilabs.io>

### CODE REVIEW

File: ament_cppcheck/ament_cppcheck/main.py [hunk 1]
```diff
@@ -126,15 +126,15 @@ def main(argv=sys.argv[1:]):
                 return 0
 
             return 188
-
```
Comment (dirk-thomas) [inline ament_cppcheck/ament_cppcheck/main.py]: Nitpick: please revert unrelated changes.

Comment (rotu) [reply ament_cppcheck/ament_cppcheck/main.py]: oops!

**Comment by rotu** [comment]
> Wouldn't it be better to mark the specific file as "skipped" to not pretend it was checked when it actually wasn't?

It does continue to find errors, just so long as they don't depend on that part of the AST. Adding `int x=1;` both inside and outside the mis-parsed `if` statement flags the missing space in both places.

**Comment by dirk-thomas** [comment]
Thanks for details. Sounds reasonable then.

**Comment by dirk-thomas** [comment]
(Will merge without CI since ci.ros2.org has issues at the moment and this option is available in `cppcheck` on Bionic so I don't expect any regressions.)
